src/message.ts

```diff
@@ -160,6 +160,29 @@ export interface MsgTypeMap {
   // , MessageTypeValue: MessageTypeName
 }
 
+/**
+ *
+ * Enum for AppMsgType values.
+ *
+ * @enum {number}
+ * @property {number} TEXT                    - 1      for TEXT
```

Comment:
We should just let users know AppMsgType.IMG, without the exact number.

When we use Wechaty, we should just use AppMsgType.IMG instead of `2`

Reply:
After I thought more if any want to know the number for more development, such as ipad protocol or xposed, maybe we should tell them the wechat raw number, So I add both `AppMsgType .TEXT`  and `1` here.

Reply:
I'll not argue with you about that, It's ok.

But I'd like to assume that the other protocols are different with the Web.